### ISSUE TITLE
Migrate "The Promise of Business Blockchain Technologies" from EdX to Markdown

### ISSUE THREAD

**Issue by tkuhrt** [hyperledger/education#46]
As part of our effort to create a second revision of the LFS171x EdX course, migrate the chapter "The Promise of Business Blockchain Technologies" to a markdown file located at `./LFS171x/docs/the-promise-of-business-blockchain-technologies.md` in the `import-edx-content` branch.

**Comment by atc-vince** [comment]
Started working on this section...

**Comment by atc-vince** [comment]
Completed draft.  Feedback welcome.

**Comment by tkuhrt** [comment]
Hi, Vince ( @atc-vince ). I could not find a Pull Request for the changes you made. I see a number of commits, but they do not have the appropriate DCO sign-off. Can you tell me how you got these changes committed to the branch? I am afraid we are going to have to undo them so that we can get the appropriate DCO sign-offs.

**Comment by atc-vince** [comment]
Hi Tracy (@tkuhrt), I wasn’t aware of the workflow process at the time when the commits were made. I tried to revert the commits, but the write access to the file had been already been deactivated. I’m good with starting over, and was waiting on guidance on how to properly “wipe the slate”.

**Comment by tkuhrt** [comment]
Hi, Vince (@atc-vince). We did wipe the commits yesterday; however, we did maintain a copy of the work that you have already done so that we could have you re-submit the file. I was going to put together a gist with information on doing just that. I will send that your way hopefully by the end of today. I appreciate your help with this, and I am sorry for the confusion/frustration that this may have caused. I am glad that you are still willing to work on this. Look for more information soon.

**Comment by atc-vince** [comment]
Hi Tracy (@tkuhrt) thanks for clearing the commits. I have a copy of the file so no need to resend. I’ll work on it (properly!) tonight. ;-)

Additionally, I can also assist with updating the documentation so that new WG Members can avoid repeating this scenario in the future. Apologies as well for the inconvenience.

**Comment by tkuhrt** [comment]
Hi, Vince (@atc-vince). Glad to hear. I have created a [Gist on how to contribute to the education repo](https://gist.github.com/tkuhrt/10211ae0a26a91a8c030d00344f7d11b). I would love your feedback on this Gist so that we can improve it for others who want to contribute.

Thanks alot for your help on this work.

**Comment by atc-vince** [comment]
Hi Tracy (@tkuhrt) thanks for the gist. Will run through it and provide feedback (if any).
